lib/web/fotorama/fotorama.js

```diff
@@ -1140,7 +1140,24 @@ fotoramaVersion = '4.6.4';
 
     function addEvent(el, e, fn, bool) {
         if (!e) return;
-        el.addEventListener ? el.addEventListener(e, fn, !!bool) : el.attachEvent('on' + e, fn);
+        let supportsPassive = false;
+        try {
+            const options = {
+                get passive() {
+                    supportsPassive = true;
+                },
+            };
+
+            document.addEventListener('test', null, options);
+            document.removeEventListener('test', null, options);
+        } catch (err) {}
+
+        const options = (supportsPassive) ? {
+            passive: true
+        } : {
+            passive: !!bool
+        };
```

Comment:
The old code used `!!bool` as the third argument, which is equivalent to `{ capture: !!bool }` not `{ passive: !!bool }`. Is this change intentional?
```suggestion
        } : !!bool;
```

```suggestion
        } : {
            capture: !!bool,
        };
```